.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,11 +3,17 @@ name: Bug Report
 about: Generic template for reporting bugs.
 labels: '🐛 bug'
 title: '🐛 Bug Report'
-assignees: ''
+assignees: '@visormatt, @nikgavalas'
```

Comment:
I'll have to test this piece, but we can also auto-assign it so that we get immediate 👀  on new issues as they are created.

Reply:
That's cool!